doc/index.rst

```diff
@@ -134,6 +134,7 @@ PennyLane is **free** and **open source**, released under the Apache License, Ve
    installing
    plugins
    research
+   Get Help<https://discuss.pennylane.ai/>
```

Comment:
```suggestion
   Get help<https://discuss.pennylane.ai/>
```

Reply:
Alternative suggestions:
`Help`
`Help forums`
@josh146 any preferences?

Reply:
`Questions and answers`
`Discussion`
`Ask for help`

Reply:
I quite like `Ask for help`?

Reply:
Just `Help` or `Ask for help` are good.